### DISCUSSION TITLE
Changing the colour of certain words in text

### DISCUSSION THREAD

**Discussion by RealKiryl** [TomSchimansky/CustomTkinter#1288] [Q&A] [answered]
Hi guys. I've encountered an obstacle. How do I change the colour of certain words in my text which is supposed to be displayed via CTk UI?
Does anyone has an example of how the code may look like?

**Comment by abhaymakes** [comment, marked as answer]
What you're looking for is the tag feature in tkinter/customtkinter. You can find more about it [here](https://tkdocs.com/tutorial/text.html#tags)

Here is an example code:
```python
from tkinter import *

root = Tk() 


text = Text(root)  

text.insert(INSERT, "Hello, everyone!") 


text.pack(expand=1, fill=BOTH)  

text.tag_add("start", "1.0", "1.5")  

text.tag_config("start",foreground="red") 

root.mainloop()
```

**Reply by RealKiryl** [reply]
Thank you for your help! It works as described.
